benchmarks/gravity_prem/gnuplot_script_PREM_density_profile

```diff
@@ -0,0 +1,67 @@
+# -------------------------------------------------------------------------------------
```

Comment:
We usually use the extension `plt` for gnuplot scripts. Please rename all gnuplot script files, e.g. this one into `PREM_density_profile.plt`. While this is not a universal standard it makes finding files easier for us.